lib/cookie.js

```diff
@@ -49,6 +50,7 @@ class Cookie {
     if (attrs.expires) header += '; expires=' + attrs.expires.toUTCString();
     if (attrs.domain) header += '; domain=' + attrs.domain;
     if (attrs.sameSite) header += '; samesite=' + (attrs.sameSite === true ? 'strict' : attrs.sameSite.toLowerCase());
+    if (attrs.sameSite === false) header += '; samesite=none';
```

Comment:
this will breaking change, we should keep false and set nothing.

Reply:
If any breaking change is unacceptable, then all falsy value (false, null, undefined, 0, '', NaN)  should be treated as a no-op.